generic/BRGenericHandlers.h

```diff
@@ -140,6 +148,10 @@ extern "C" {
             BRGenericFeeBasisIsEqual feeBasisIsEqual;
             BRGenericFeeBasisFree free;
         } feebasis;
+
+        struct {
+            BRGenericNetworkCreateAddress createAddress;
+        } network;
```

Comment:
Nice, Ripple, Stellar, etc only need to create addresses; not networks to be created/released.

Reply:
Although I think there needs to be `addressRelease` and ('createAddress' -> 'addressCreate' or 'networkCreateAddress')

Reply:
Fixed in latest checkin